packages/core/src/internal/io.js

```diff
@@ -71,46 +71,51 @@ export function race(effects) {
   return makeEffect(effectTypes.RACE, effects)
 }
 
-function getFnCallDesc(meth, fn, args) {
+const callFnEffects = ['call', 'apply', 'cps', 'fork', 'spawn']
+
+function getFnCallDescriptor(fnDescriptor, args) {
   if (process.env.NODE_ENV !== 'production') {
-    check(fn, is.notUndef, `${meth}: argument fn is undefined`)
+    check(fnDescriptor, is.notUndef, `${callFnEffects.join('/')}: argument fn is undefined`)
```

Comment:
imho message  `call/apply/cps/fork/spawn: argument fn is undefined` seems confusing as user calls a method, not all of them

Reply:
right, gonna move validation to the effect creators from get `getFnCallDescriptor` itself